pkgs/development/rocm-modules/5/clr/default.nix

```diff
@@ -88,6 +88,11 @@ in stdenv.mkDerivation (finalAttrs: {
     "-DCMAKE_INSTALL_LIBDIR=lib"
   ];
 
+  patches = [
+    ./add-missing-operators.patch
```

Comment:
I think that you should download the patch directly from github using fetchpatch istead of including it nixpkgs.